MapboxCoreNavigation/CLLocation.swift

```diff
@@ -71,7 +71,7 @@ extension CLLocation {
         guard let closestCoordinate = Polyline(routeStep.coordinates!).closestCoordinate(to: coordinate) else {
             return false
         }
-        return closestCoordinate.distance < maximumDistance
+        return closestCoordinate.distance <= maximumDistance
```

Comment:
What spurred this change?

Reply:
@bsudekum During dead reckoning test, we encountered reroute because the `closestCoordinate.distance` was, `15` -- equal to the `maximumDistance`. Moreover, the documentation for the function indicates, the maximum distance is inclusive.

>  /**
      Returns a Boolean value indicating whether the receiver is within a given distance of a route step, inclusive.
      */

https://github.com/mapbox/mapbox-navigation-ios/pull/1218/files/732ff8b42b477915ef36eafa78d8cf849ff1fb6a#diff-dfd8aa8fe700af70004116a3a54c9d84L67